app/src/main/res/layout/fragment_calendar_non_colored.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
These two files are exactly the same and can just be `fragment_calendar.xml`

Reply:
👍